src/RA_Dlg_MemBookmark.cpp

```diff
@@ -310,6 +311,7 @@ INT_PTR Dlg_MemBookmark::MemBookmarkDialogProc(HWND hDlg, UINT uMsg, WPARAM wPar
                         if (nSelect == -1)
                             break;
                     }
+#pragma warning(suppress: 26454) // io.5; arithmetic overflow
                     else if (((LPNMHDR)lParam)->code == NM_DBLCLK)
```

Comment:
You can't put C++ attributes (at least currently) before an `else if`, so it uses the warning code instead.